Cornelica/scenario/010.txt

```diff
@@ -182,45 +182,45 @@
 
 @huki b=7 id=12
 @bs f=リリィ@困惑 op=困惑 body=-1 face=0 x=350 y=50 t=200
-[Lily] "... After all ... ... when I was hugged a little while ago ...."
+[Lily]"...After all...when you was hugged a little while ago...."
 
 @huki id=11 b=2
 @bs f=アル@困り_頬 op=ハテナ body=-1 face=0 x=-350 y=50 t=200
 @wait t=600
 
 @bs f=リリィ@焦り body=-1 face=0 x=350 y=50 t=200
 
-[Lily] "Uh ... ... us, Succubi are always exuding pheromones \ C [0] that is \ C [2]" charm ""
+[Lily]"Uh... us, Succubi have magical power called \C[2]charm\C[2]"
```

Comment:
That last `\C[2]` probably should be a 0 instead.

Reply:
Yep, it's important too... arrgg....